app/routes/events/components/JoinEventForm.js

```diff
@@ -279,102 +279,117 @@ class JoinEventForm extends Component<Props> {
                   </Link>
                 </div>
               )}
-              {formOpen && (
-                <Flex column>
-                  <Form
-                    onSubmit={this.submitWithType(
-                      handleSubmit,
-                      feedbackName,
-                      registrationType
-                    )}
-                  >
-                    {showCaptcha && (
+              {!disabledForUser &&
```

Comment:
It's probably better to do the same as for surveys to avoid making `JoinEventForm` bigger